src/kit.ts

```diff
@@ -130,7 +130,9 @@ async function getClangVersion(binPath: string): Promise<ClangVersion|null> {
     return null;
   }
   const lines = exec.stderr.split('\n');
-  const version_re = /^(?:Apple LLVM|.*clang) version ([^\s-]+)(?:[\s-]|$)/;
+  const versionWord: string = localize("version.word", "version");
```

Comment:
I think we need to check for both the English and the localized word.  Perhaps we do multiple passes on this?  Does the regex stuff support Unicode characters too?

Reply:
I just verified that this regexp functionality works fine with a string like 版本, so we can count on that.
And yes, good catch, check for both.
I also investigated the LANG environment variables (to enforce that we get english output when running grcc/clang with -v) but since it's not straightforward for me to reproduce and you also said you had problems in the past, including one customer, I think checking twice with 2 regexp is acceptable.